torch/lib/THD/base/data_channels/DataChannelGloo.cpp

```diff
@@ -187,35 +188,35 @@ void DataChannelGloo::allReduceT(thpp::Tensor& t, THDReduceOp operation,
 
 void DataChannelGloo::allReduce(thpp::Tensor& data, THDReduceOp operation,
                                 THDGroup group_id) {
+  RETURN_IF_NOT_IN_GROUP(_)
   GENERATE_ALL_TYPES(data.type(), allReduceT, data, operation, group_id)
 }
 
 
 // XXX: `reduce` is not supported by Gloo yet.
 void DataChannelGloo::reduce(thpp::Tensor& data, THDReduceOp operation,
                              rank_type dst_rank, THDGroup group_id) {
-  throw std::runtime_error("DataChannelGloo doesn't support reduce");
+  throw std::runtime_error("DataChannelGloo does not support reduce");
 }
 
 
 template<typename T>
 void DataChannelGloo::broadcastT(thpp::Tensor& data, rank_type src_rank,
-                                THDGroup group_id) {
-  auto& store = getStore();
+                                 THDGroup group_id) {
   RETURN_IF_NOT_IN_GROUP(group_rank)
```

Comment:
It seems a bit weird that you always use this macro outside of the `T` functions, except in here 😕 You're not even using the result.